src/modern_treasury/types/invoice_update_params.py

```diff
@@ -0,0 +1,144 @@
+    if it exists in the test environment.
+    """
+
+    object: Required[str]
+
+    updated_at: Required[Annotated[Union[str, datetime], PropertyInfo(format="iso8601")]]
+
+
+ContactDetails = ContactDetail
+"""This type is deprecated and will be removed in a future release.
+
+Please use ContactDetail instead.
+"""
+
+
+class CounterpartyBillingAddress(TypedDict, total=False):
+    country: Required[str]
+    """Country code conforms to [ISO 3166-1 alpha-2]"""
+
+    line1: Required[str]
+
+    locality: Required[str]
+    """Locality or City."""
+
+    postal_code: Required[str]
+    """The postal code of the address."""
+
+    region: Required[str]
+    """Region or State."""
+
+    line2: str
+
+
+class CounterpartyShippingAddress(TypedDict, total=False):
+    country: Required[str]
+    """Country code conforms to [ISO 3166-1 alpha-2]"""
+
+    line1: Required[str]
+
+    locality: Required[str]
+    """Locality or City."""
+
+    postal_code: Required[str]
+    """The postal code of the address."""
+
+    region: Required[str]
+    """Region or State."""
+
+    line2: str
+
+
+class InvoicerAddress(TypedDict, total=False):
+    country: Required[str]
+    """Country code conforms to [ISO 3166-1 alpha-2]"""
+
+    line1: Required[str]
+
+    locality: Required[str]
+    """Locality or City."""
+
+    postal_code: Required[str]
+    """The postal code of the address."""
+
+    region: Required[str]
+    """Region or State."""
+
+    line2: str
+
+
+class InvoiceUpdateParams(TypedDict, total=False):
+    contact_details: List[ContactDetail]
+    """The invoicer's contact details displayed at the top of the invoice."""
+
+    counterparty_billing_address: Optional[CounterpartyBillingAddress]
+    """The counterparty's billing address."""
+
+    counterparty_id: str
+    """The ID of the counterparty receiving the invoice."""
+
+    counterparty_shipping_address: Optional[CounterpartyShippingAddress]
+    """The counterparty's shipping address where physical goods should be delivered."""
+
+    currency: shared_params.Currency
+    """Currency that the invoice is denominated in. Defaults to `USD` if not provided."""
+
+    description: str
+    """A free-form description of the invoice."""
+
+    due_date: Annotated[Union[str, datetime], PropertyInfo(format="iso8601")]
+    """A future date by when the invoice needs to be paid."""
+
+    include_payment_ui: bool
+    """
+    When opening an invoice, whether to show the embedded payment UI with the
+    invoice. Default true.
+    """
+
+    invoicer_address: Optional[InvoicerAddress]
+    """The invoice issuer's business address."""
+
+    originating_account_id: str
+    """The ID of the internal account the invoice should be paid to."""
+
+    status: str
+    """
+    Invoice status must be updated in a `PATCH` request that does not modify any
+    other invoice attributes. Valid state transitions are `draft` to `unpaid`,
+    `draft` or `unpaid` to `void` and `unpaid` to `closed`.
```

Comment:
@cleb11 A recent change to this description wasn't picked up - I think I need to do the OpenAPI sync first